<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainFRM</class>
 <widget class="QMainWindow" name="mainFRM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>660</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>mainFRM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>Connexion</string>
         </attribute>
         <widget class="QWidget" name="verticalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>571</width>
            <height>301</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLineEdit" name="loginTXT">
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>Identifiant</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="passwordTXT">
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>Mot de passe</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="connexionBTN">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Connexion</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_6">
         <attribute name="title">
          <string>Inscription</string>
         </attribute>
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>571</width>
            <height>301</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLineEdit" name="register_loginTXT">
             <property name="placeholderText">
              <string>Identifiant</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="register_passwordTXT">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>Mot de passe</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="register_passwordBISTXT">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>Confirmation du mot de passe</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="registerBTN">
             <property name="text">
              <string>S'inscrire</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuParam_tre">
    <property name="title">
     <string>Param√®tre</string>
    </property>
    <addaction name="actionChanger_HOST"/>
    <addaction name="actionChanger_PORT"/>
   </widget>
   <addaction name="menuParam_tre"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionChanger_HOST">
   <property name="text">
    <string>Changer HOST</string>
   </property>
  </action>
  <action name="actionChanger_PORT">
   <property name="text">
    <string>Changer PORT</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
